Add documentation for new broker configuration fields (#5215)
EnMasseProject/enmasse: documentation/modules/ref-standard-infra-config-fields.adoc

|`broker.connectorWorkerThreads` |Specifies the number of worker threads of the connection to the router.
|`broker.storageClassName` |Specifies what storage class to use for the persistent volume for the broker.
|`broker.updatePersistentVolumeClaim` |If the persistent volume supports resizing, setting this value to `true` allows the broker storage to be resized.

> |`broker.treatRejectAsUnmodifiedDeliveryFailed` | Treat rejected delivery outcome as modified delivery failed. This causes the message to be re-sent to the consumer by default. Default is true.

Reviewer: ```suggestion
|`broker.treatRejectAsUnmodifiedDeliveryFailed` | Treat rejected delivery outcome as modified delivery failed. This causes the message to be re-sent to the consumer by default. The default value is `true`.
```

> |`broker.useModifiedForTransientDeliveryErrors` | Respond with modified for transient delivery errors to allow sender to retry. Default is true.

Reviewer: ```suggestion
|`broker.useModifiedForTransientDeliveryErrors` | Respond with modified for transient delivery errors to allow sender to retry. The default value is `true`.
```

> |`broker.minLargeMessageSize` | Specify the minimum size of a message for it to be treated as a large message. A large message is always paged to disk with a reference in the journal. Default is -1 (never page).

Reviewer: ```suggestion
|`broker.minLargeMessageSize` | Specifies the minimum size of a message for it to be treated as a large message. A large message is always paged to disk with a reference in the journal.The default value is `-1` (never page).
```
